rkyv_derive/src/archive/struct.rs

```diff
@@ -176,11 +176,18 @@ fn generate_archived_type(
             ..
         } = field;
 
+        let field_name = ident.as_ref();
+        let field_doc = format!(
+            "The archived counterpart of [`{}:{}`]",
+            name,
+            field_name.map_or("unnamed".to_string(), |s| s.to_string())
```

Comment:
unnamed fields should probably be better handled with their indices, but I'll continue after a validation.

Reply:
Thanks! Should we have testing on those documentations ?

It's a tiny bit tricky to test because we should probably create a new library dedicated to testing which should deny missing docs 🤔 ; maybe better to leave for another PR?